sr_unix/mdefsp.h

```diff
@@ -183,8 +184,13 @@ typedef char  mach_inst;	/* machine instruction */
 #  endif
 #endif
 
-#if defined(__ARM_ARCH_7A__)
-#  define __armv7l__
+#if __arm__
+#  if __ARM_ARCH_6__
+#    define __armv6l__
+#  elif __ARM_ARCH_7A__
+#    define __armv7l__
+#  endif
```

Comment:
Should there be an else here that raises an error of an unsupported ARM type?

Reply:
Done.